source/prerequisites/postgres.txt

```diff
@@ -1,13 +1,84 @@
 .. _rm-prereq-postgres:
 
-================================
-Postgres Migration Prerequisites
-================================
+==================================
```

Comment:
[Info Typing]

Task page titles should be verb phrases.

Reply:
Will ask Tom about a preferred name for these pages that is more verb driven and update accordingly.

Reply:
Sounds good, thanks Ian!